app/panel/components/Panel.jsx

```diff
@@ -369,6 +384,30 @@ class Panel extends React.Component {
 				handleSignInClick={this._handlePromoSignInClick}
 				handleSubscribeClick={this._handlePromoSubscribeClick}
 				handleXClick={this._handlePromoXClick}
+				show
+			/>
+		);
+	}
+
+	/**
+	 * @returns {null|JSX}
+	 * @private
+	 * Renders the Insights promo modal if the user is not already an Insights subscriber
```

Comment:
Looks like this comment needs to be updated

Reply:
Updated comment!